extensions/wikia/PortableInfobox/tests/nodes/NodeDataTest.php

```diff
@@ -46,11 +46,12 @@ public function sourceDataProvider() {
 	 * @param $params
 	 * @param $expected
 	 */
-	public function testSourceLabel( $markup, $params, $expected ) {
-		$node = \Wikia\PortableInfobox\Parser\Nodes\NodeFactory::newFromXML( $markup, $params );
-
-		$this->assertEquals( $expected, $node->getSourceLabel() );
-	}
+	// TODO
+	//public function testSourceLabel( $markup, $params, $expected ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)